src/pages/authPage/SignInForm.tsx

```diff
@@ -16,6 +17,21 @@ const LogInForm = () => {
 
   const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
+    try {
+        await userService.signInWithEmailAndPassword(email, password);
+        alert("Sign in successful!")
```

Comment:
Is this alert just temporary? Since it will redirect to the main page when sign in is succesful?

Reply:
Yea its just temp for now